docs/manual/scala/guide/build/MultipleBuilds.md

```diff
@@ -0,0 +1,56 @@
+# Splitting a system into multiple builds
+
+For a small system maintained by a single team, it's fine to have all your services in one build. Doing it that way makes it really easy to run all your services with the `runAll` task, as we'll see later in the [[Running Services|RunningServices]] section of this manual.
+
+If you have multiple teams, though, then as described already in [[Lagom build concepts|BuildConcepts]], we recommend splitting your system into multiple builds.
+
+If you aren't concerned with scaling to multiple teams yet, feel free to skip this section for now.
+
+## Publishing services
+
+Even with multiple builds, you will still often want to run your services together in development.  Lagom allows importing services published from one build into another build.
+
+Suppose you have a `hello` service that you want to publish and import into another build.  You can publish this to your local repository by running `mvn install` if using Maven, or by running `publishLocal` if using sbt.  This is the simplest way to publish a service, however it means every developer that wants to run a build that imports the service will need publish it to their own repository themselves, and they'll need to do that for each version that they want to import.
+
```

Comment:
I think that's OK, since this is documentation for incorporating some other Lagom project into the current build. That other project might be a Java/Maven project, and this one Scala/sbt.

I know that we don't currently support direct interoperation between Scala and Java Lagom projects. Is that a requirement for 1.3.0 final?